### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_Non-Text Contrast: Color contrast ratio of the fusion icon is less than 3:1

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#175]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#A11yWCAG2.1;#WCAG1.4.11;#DesktopWeb;#Win11;#ChromiumEdge

### Environment Details:
Application Name: 30days 
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/ 
2.	Home page will open then tab to "fusion" present in header section.
3.	Start the color contrast analyzer tool
4.	Observe color contrast ratio of “fusion” icon is less than 3:1
### Actual:
Color contrast ration of “fusion” icon is less than 3:1 i.e the ratio is 1.003:1.
### Expected:
Color contrast ratio of “fusion icon” should be equal or greater than 3:1
### User Impact:
Low vision user will not be able to understand the icon.

### Attachment:
<img width="959" alt="2_A11y_ WCP-CELAP3 30days_Welcome to 30 Days of Learning_Non-Text Contrast Color contrast ratio of the fusion icon is less than 31" src="https://user-images.githubusercontent.com/85863895/209336588-92196137-c21a-495f-b194-e49fd10e8103.png">

**Comment by nitya** [comment]
Fusion and other roadmap icons are no longer in the navbar

**Comment by Raghu-addepalli** [comment]
![image](https://user-images.githubusercontent.com/91586743/220269354-d578a6cf-e94c-4c72-b427-98ca8b3349c0.png)
verified the issue in below https://microsoft.github.io/30daysof/ environment the controls causing the issue are not available in environment hence closing the bug and added attachment.
Githubtags:  #Closed;
